lib/features/game/views/widgets.dart

```diff
@@ -54,17 +54,21 @@ class AnswerButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Center(
-      child: ElevatedButton(
-        onPressed: () => context.read<GamePageBloc>().add(AnswerQuestion(buttonState.index)),
-        child: Text(buttonState.answer),
-        style: ElevatedButton.styleFrom(
-          primary: getButtonColor(),
-          alignment: Alignment.center,
-          shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(10.0)),
-          elevation: 5,
-          padding: const EdgeInsets.symmetric(horizontal: 16),
-          textStyle: TextStyle(color: KColors.basedBlackColor, fontSize: 15),
-          minimumSize: Size(140, 120),
+      child: SizedBox(
+        width: 120,
```

Comment:
hardcoded size is a bad idea. Use some grid magic to make them same size but different for various screen sizes

Reply:
done